From 32beef3ac2c308afcbc8ded14ebfc3c8a72d1a80 Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Sun, 6 May 2018 11:26:25 +0000
Subject: [PATCH] timeoutdialog.cpp: include QIcon, QStyle explicitely

5.11 Qt headers do not indirectly include QIcon, QStyle resulting in compile
time errors like:

| timeoutdialog.cpp:32:25: error: invalid use of incomplete type 'class QStyle'

Upstream-Status: Submitted [https://github.com/lxqt/lxqt-config/pull/151]

Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 lxqt-config-monitor/timeoutdialog.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lxqt-config-monitor/timeoutdialog.cpp b/lxqt-config-monitor/timeoutdialog.cpp
index 55302a9..0862447 100644
--- a/lxqt-config-monitor/timeoutdialog.cpp
+++ b/lxqt-config-monitor/timeoutdialog.cpp
@@ -20,6 +20,9 @@
 
 #define TIMER_DURATION 10
 
+#include <QIcon>
+#include <QStyle>
+
 #include "timeoutdialog.h"
 
 TimeoutDialog::TimeoutDialog(QWidget* parent, Qt::WindowFlags f) :
-- 
2.13.6

